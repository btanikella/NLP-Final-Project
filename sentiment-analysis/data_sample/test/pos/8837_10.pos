DT JJS NN NN FW VBP RB VBN
PRP VBD PRP IN DT NNP  VBG IN JJ NN  IN DT JJ JJ NN WRB PRP VBP DT NN IN DT JJ NN : JJ TO VB PRP VBD IN DT NN :
