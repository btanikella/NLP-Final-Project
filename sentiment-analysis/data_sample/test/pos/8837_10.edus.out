Sentence #1 (9 tokens):
The best horror/sci-fi movie i have ever seen.
[Text=The CharacterOffsetBegin=4 CharacterOffsetEnd=7 PartOfSpeech=DT] [Text=best CharacterOffsetBegin=8 CharacterOffsetEnd=12 PartOfSpeech=JJS] [Text=horror/sci-fi CharacterOffsetBegin=13 CharacterOffsetEnd=26 PartOfSpeech=NN] [Text=movie CharacterOffsetBegin=27 CharacterOffsetEnd=32 PartOfSpeech=NN] [Text=i CharacterOffsetBegin=33 CharacterOffsetEnd=34 PartOfSpeech=FW] [Text=have CharacterOffsetBegin=35 CharacterOffsetEnd=39 PartOfSpeech=VBP] [Text=ever CharacterOffsetBegin=40 CharacterOffsetEnd=44 PartOfSpeech=RB] [Text=seen CharacterOffsetBegin=45 CharacterOffsetEnd=49 PartOfSpeech=VBN] [Text=. CharacterOffsetBegin=49 CharacterOffsetEnd=50 PartOfSpeech=.] 
root(ROOT-0, seen-8)
det(i-5, The-1)
amod(i-5, best-2)
nn(i-5, horror/sci-fi-3)
nn(i-5, movie-4)
nsubj(seen-8, i-5)
aux(seen-8, have-6)
advmod(seen-8, ever-7)
punct(seen-8, .-9)

Sentence #2 (36 tokens):
I was myself in the Arctic, working for Canadian government , in a small northern station when I see this movie for the first time; needless to say I was in the mood...
[Text=I CharacterOffsetBegin=55 CharacterOffsetEnd=56 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=57 CharacterOffsetEnd=60 PartOfSpeech=VBD] [Text=myself CharacterOffsetBegin=61 CharacterOffsetEnd=67 PartOfSpeech=PRP] [Text=in CharacterOffsetBegin=68 CharacterOffsetEnd=70 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=71 CharacterOffsetEnd=74 PartOfSpeech=DT] [Text=Arctic CharacterOffsetBegin=75 CharacterOffsetEnd=81 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=81 CharacterOffsetEnd=82 PartOfSpeech=,] [Text=working CharacterOffsetBegin=83 CharacterOffsetEnd=90 PartOfSpeech=VBG] [Text=for CharacterOffsetBegin=91 CharacterOffsetEnd=94 PartOfSpeech=IN] [Text=Canadian CharacterOffsetBegin=95 CharacterOffsetEnd=103 PartOfSpeech=JJ] [Text=government CharacterOffsetBegin=104 CharacterOffsetEnd=114 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=115 CharacterOffsetEnd=116 PartOfSpeech=,] [Text=in CharacterOffsetBegin=117 CharacterOffsetEnd=119 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=120 CharacterOffsetEnd=121 PartOfSpeech=DT] [Text=small CharacterOffsetBegin=122 CharacterOffsetEnd=127 PartOfSpeech=JJ] [Text=northern CharacterOffsetBegin=128 CharacterOffsetEnd=136 PartOfSpeech=JJ] [Text=station CharacterOffsetBegin=137 CharacterOffsetEnd=144 PartOfSpeech=NN] [Text=when CharacterOffsetBegin=145 CharacterOffsetEnd=149 PartOfSpeech=WRB] [Text=I CharacterOffsetBegin=150 CharacterOffsetEnd=151 PartOfSpeech=PRP] [Text=see CharacterOffsetBegin=152 CharacterOffsetEnd=155 PartOfSpeech=VBP] [Text=this CharacterOffsetBegin=156 CharacterOffsetEnd=160 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=161 CharacterOffsetEnd=166 PartOfSpeech=NN] [Text=for CharacterOffsetBegin=167 CharacterOffsetEnd=170 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=171 CharacterOffsetEnd=174 PartOfSpeech=DT] [Text=first CharacterOffsetBegin=175 CharacterOffsetEnd=180 PartOfSpeech=JJ] [Text=time CharacterOffsetBegin=181 CharacterOffsetEnd=185 PartOfSpeech=NN] [Text=; CharacterOffsetBegin=185 CharacterOffsetEnd=186 PartOfSpeech=:] [Text=needless CharacterOffsetBegin=187 CharacterOffsetEnd=195 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=196 CharacterOffsetEnd=198 PartOfSpeech=TO] [Text=say CharacterOffsetBegin=199 CharacterOffsetEnd=202 PartOfSpeech=VB] [Text=I CharacterOffsetBegin=203 CharacterOffsetEnd=204 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=205 CharacterOffsetEnd=208 PartOfSpeech=VBD] [Text=in CharacterOffsetBegin=209 CharacterOffsetEnd=211 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=212 CharacterOffsetEnd=215 PartOfSpeech=DT] [Text=mood CharacterOffsetBegin=216 CharacterOffsetEnd=220 PartOfSpeech=NN] [Text=... CharacterOffsetBegin=220 CharacterOffsetEnd=223 PartOfSpeech=:] 
root(ROOT-0, was-2)
nsubj(was-2, I-1)
nsubj(was-2, myself-3)
prep(myself-3, in-4)
det(Arctic-6, the-5)
pobj(in-4, Arctic-6)
punct(myself-3, ,-7)
partmod(myself-3, working-8)
prep(working-8, for-9)
amod(government-11, Canadian-10)
pobj(for-9, government-11)
punct(myself-3, ,-12)
prep(myself-3, in-13)
det(station-17, a-14)
amod(station-17, small-15)
amod(station-17, northern-16)
pobj(in-13, station-17)
advmod(see-20, when-18)
nsubj(see-20, I-19)
rcmod(myself-3, see-20)
det(movie-22, this-21)
dep(see-20, movie-22)
prep(movie-22, for-23)
det(time-26, the-24)
amod(time-26, first-25)
pobj(for-23, time-26)
punct(was-2, ;-27)
dep(was-2, needless-28)
aux(say-30, to-29)
infmod(needless-28, say-30)
nsubj(was-32, I-31)
ccomp(say-30, was-32)
prep(was-32, in-33)
det(mood-35, the-34)
pobj(in-33, mood-35)
punct(was-2, ...-36)

